backend/README.md

```diff
@@ -48,4 +48,14 @@ process.env.FOOS_CHANNEL_NAME
 ```
 - these variables are used by the slackbot to announce the results of the matches. You can obtain the bot Token on slack settings page and then select the channel you'd like to deploy the bot on. 
 
-
+```
+process.env.MATCH_RESULT_PREFIX_SUFFIX_CONFIG
+```
+- used to configure prefixes and suffixes of the matches results messages (see above) based on the players on the winning team.
+- the app expects quadruplets of player1, player2, prefix and suffix to be encoded in the environment variable.
+- use `,` to separate the values within a quadruplet (`player1,player2,prefix,suffix`) and `;` to separate the quadruplets (`quadruplet1;quadruplet2`). Note: the order of players in the quadruplet does not matter. 
```

Comment:
also, i'd consider making it more generic - like "<numberOfRequeredPlayers>, <names>, prefix, sufix", eg "2Pepa, Tomy,Stego,michalvenos,good idea,slovakia" "1,Pepa,Tomy,Stego, heyhey, one slovak guy on a team"

this might be a bit too much though.... :)

Reply:
Sorry, I don't have the time for that anymore.

Reply:
I AM THE CODE OWNER, OBEY ME

Reply:
JK ;)